alexandria/oidc_auth/authentication.py

```diff
@@ -82,3 +83,33 @@ def cached_request(self, method, token, cache_prefix):
             func,
             timeout=self.OIDC_BEARER_TOKEN_REVALIDATION_TIME,
         )
+
+
+class DummyAuthenticationBackend(AlexandriaAuthenticationBackend):
+    def __init__(self, *args, **kwargs):  # pragma: no cover
+        super().__init__(*args, **kwargs)
+        if not settings.DEBUG:
+            raise ImproperlyConfigured(
+                "Dummy Auth Backend can only be used in DEBUG mode!"
+            )
+
+    def get_introspection(self, access_token, id_token, payload):  # pragma: no cover
```

Comment:
If I'm not mistaken, this is only called from within `get_userinfo_or_introspection()`. So there is no need to override it.

Reply:
That's what it seemed to me as well, but given that it's a public method...